compiler/rustc_resolve/src/lib.rs

```diff
@@ -1506,6 +1517,9 @@ impl<'a> Resolver<'a> {
     pub fn resolve_crate(&mut self, krate: &Crate) {
         self.session.time("resolve_crate", || {
             self.session.time("finalize_imports", || ImportResolver { r: self }.finalize_imports());
+            self.session.time("resolve_access_levels", || {
+                PrivacyVisitor::compute_access_levels(self, krate)
```

Comment:
```suggestion
                AccessLevelsVisitor::compute_access_levels(self, krate)
```
(Missing part of https://github.com/rust-lang/rust/pull/87487#discussion_r696577654)
Also the timer name on the line above wasn't updated.